### ISSUE TITLE
Putting out an app on Google Play with AppPresser

### ISSUE THREAD

**Issue by Supersketch** [Tmeister/wp-api-jwt-auth#193]
I feel my app is ready enough to publish to Google Play. I want to price it at £2 or £1.99. 

I'm wondering if once it's on Google Play, will I continue to need an AppPresser product membership to support it?

I'm happy to commit for a year with the cost, but wondering if I'm likely to hit problems supporting it beyond that. And if it stopped working in 12 months and customers have paid for it, will I be in some trouble with the customer/Google Playstore. 😳

Also what if the customer buys it in month 9 of my one-off 12 month membership?

Maybe I will need to add some terms and conditions in the description. 

Part of me thinks to release on Google Play as my year on AppPresser begins and deal with any questions as they arise. 

Perhaps you can help me with these details, so at least I know what I'm getting myself into!!!

**Comment by contactjavas** [comment]
Hi @Supersketch , have you found solution for your problem?
I saw Appreser has some negative reviews.

**Comment by Supersketch** [comment]
Hi I found that unless you pay the subscription the app won't stay live. And if I release on Google play then I'd feel obligated to keeping the app running. At fourth pounds a month indefinitely I decided it wasn't worth it. 
Thanks for getting in contact 👍
    On Wednesday, 1 July 2020, 16:36:25 BST, Bagus <notifications@github.com> wrote:  
 
 


Hi @Supersketch , have you found solution for your problem?
I saw Appreser has some negative reviews.
